README.md

```diff
@@ -40,9 +40,57 @@ DATABASES = {
         'PASSWORD': 'my_password',
         'ACCOUNT': 'my_account',
         # Include 'OPTIONS' if you need to specify any other
-        # snowflake.connector.connect() parameters.
+        # snowflake.connector.connect() parameters
+        # such as 'role', 'authenticator', 'private_key', etc.
+        # See the link below for a full list
         # https://docs.snowflake.com/en/user-guide/python-connector-api.html#connect
-        'OPTIONS': {},
+        'OPTIONS': {
+          # 'role': 'DJANGO_APP_RL',
+          # 'authenticator': 'https://example.okta.com',
+        },
+    },
+}
+```
+
+### Key-Pair authentication
+
+The raw keypair bytes are required to be read from a private
+key file before it can be used [with the `private_key`
+authentication config](https://docs.snowflake.com/en/developer-guide/python-connector/python-connector-example#label-python-key-pair-authn-rotation)
+in `settings.py`:
+
+```python
+import os
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+
+# Read the PEM key bytes from a local file
+with open("<path>/rsa_key.p8", "rb") as key:
```

Comment:
If possible, it would be nice not to duplicate all the code the reads and decrypts the key. Is the connector's documentation open source or accessbile to you? You could make some improvements there like removing the unused rsa/dsa imports and perhaps add the "# Pass 'None' if the key is not encrypted." comment.

At that point, we could show ```'private_key': pkb,``` in OPTIONS with a link to the connector's docs for how to obtain `pkb`.

Reply:
Yes I'll raise a request to our docs to remove the unnecessary line.

I've made the suggested change to document obtaining a 'pkb' value through a link instead of inlined code in the docs